extensions/amazon-sales/release-notes/index.md

```diff
@@ -0,0 +1,44 @@
+---
+group: extensions
+title: Amazon Sales Channel Release Notes
+---
+
+*Release notes published April 2019.*
```

Comment:
You don't really need this anymore now that we add the last modified date to each topic. It's up to you though.

Reply:
Gotcha, cutting